src/validator/GetSubgraphValidator.cpp

```diff
@@ -241,16 +245,20 @@ Status GetSubgraphValidator::toPlan() {
     return Status::OK();
 }
 
-StatusOr<std::vector<storage::cpp2::VertexProp>> GetSubgraphValidator::buildVertexProp() {
+StatusOr<GetNeighbors::VertexProps> GetSubgraphValidator::buildVertexProp() {
```

Comment:
I think you could reuse `SchemaUtil::genVertexProp` function.

Reply:
Will be refactored later, move  it to planner